certs/generate-dev-certs.sh

```diff
@@ -10,9 +10,13 @@ SSL_TRUSTSTORE=cryostat-truststore.p12
 
 SSL_KEYSTORE_PASS_FILE=keystore.pass
 
+if [ -z "$JAVA_HOME" ]; then
+    JAVA_HOME=$(readlink -f /usr/bin/java | grep -oP '.*?openjdk')
```

Comment:
Could this just use `/usr/lib/jvm/java` rather than this readlink trick? At least on Fedora and derivative distros, this location is normally present and is a symlink to whatever JDK version is configured using `alternatives`.

```
$ ls -l /usr/lib/jvm
total 96
lrwxrwxrwx. 1 root root  26 Jan 25 17:04 java -> /etc/alternatives/java_sdk
lrwxrwxrwx. 1 root root  29 Mar  1 20:11 java-11 -> /etc/alternatives/java_sdk_11
lrwxrwxrwx. 1 root root  37 Mar  1 20:11 java-11-openjdk -> /etc/alternatives/java_sdk_11_openjdk
drwxr-xr-x. 1 root root  78 Mar  1 20:09 java-11-openjdk-11.0.22.0.7-1.fc38.x86_64
lrwxrwxrwx. 1 root root  29 Jan 25 17:04 java-17 -> /etc/alternatives/java_sdk_17
lrwxrwxrwx. 1 root root  37 Jan 25 17:04 java-17-openjdk -> /etc/alternatives/java_sdk_17_openjdk
drwxr-xr-x. 1 root root 136 Jan 25 17:03 java-17-openjdk-17.0.9.0.9-3.fc38.x86_64
lrwxrwxrwx. 1 root root  29 Apr  5 11:27 java-21 -> /etc/alternatives/java_sdk_21
lrwxrwxrwx. 1 root root  37 Apr  5 11:27 java-21-openjdk -> /etc/alternatives/java_sdk_21_openjdk
drwxr-xr-x. 1 root root  78 Apr  5 11:27 java-21-openjdk-21.0.2.0.13-1.fc38.x86_64
lrwxrwxrwx. 1 root root  29 Mar 16 10:56 java-22 -> /etc/alternatives/java_sdk_22
lrwxrwxrwx. 1 root root  37 Mar 16 10:56 java-22-openjdk -> /etc/alternatives/java_sdk_22_openjdk
drwxr-xr-x. 1 root root  78 Mar 16 10:56 java-22-openjdk-22.0.0.0.36-1.rolling.fc38.x86_64
lrwxrwxrwx. 1 root root  34 Jan 25 17:04 java-openjdk -> /etc/alternatives/java_sdk_openjdk
lrwxrwxrwx. 1 root root  21 Jan 25 17:04 jre -> /etc/alternatives/jre
lrwxrwxrwx. 1 root root  24 Mar  1 20:11 jre-11 -> /etc/alternatives/jre_11
lrwxrwxrwx. 1 root root  32 Mar  1 20:11 jre-11-openjdk -> /etc/alternatives/jre_11_openjdk
lrwxrwxrwx. 1 root root  41 Jan 26 19:00 jre-11-openjdk-11.0.22.0.7-1.fc38.x86_64 -> java-11-openjdk-11.0.22.0.7-1.fc38.x86_64
lrwxrwxrwx. 1 root root  24 Jan 25 17:04 jre-17 -> /etc/alternatives/jre_17
lrwxrwxrwx. 1 root root  32 Jan 25 17:04 jre-17-openjdk -> /etc/alternatives/jre_17_openjdk
lrwxrwxrwx. 1 root root  40 Dec 15 19:00 jre-17-openjdk-17.0.9.0.9-3.fc38.x86_64 -> java-17-openjdk-17.0.9.0.9-3.fc38.x86_64
lrwxrwxrwx. 1 root root  24 Apr  5 11:27 jre-21 -> /etc/alternatives/jre_21
lrwxrwxrwx. 1 root root  32 Apr  5 11:27 jre-21-openjdk -> /etc/alternatives/jre_21_openjdk
lrwxrwxrwx. 1 root root  41 Jan 19 19:00 jre-21-openjdk-21.0.2.0.13-1.fc38.x86_64 -> java-21-openjdk-21.0.2.0.13-1.fc38.x86_64
lrwxrwxrwx. 1 root root  24 Mar 16 10:56 jre-22 -> /etc/alternatives/jre_22
lrwxrwxrwx. 1 root root  32 Mar 16 10:56 jre-22-openjdk -> /etc/alternatives/jre_22_openjdk
lrwxrwxrwx. 1 root root  49 Feb 18 19:00 jre-22-openjdk-22.0.0.0.36-1.rolling.fc38.x86_64 -> java-22-openjdk-22.0.0.0.36-1.rolling.fc38.x86_64
lrwxrwxrwx. 1 root root  29 Jan 25 17:04 jre-openjdk -> /etc/alternatives/jre_openjdk

$ ls -l /etc/alternatives/java_sdk
lrwxrwxrwx. 1 root root 53 Apr  5 11:48 /etc/alternatives/java_sdk -> /usr/lib/jvm/java-17-openjdk-17.0.9.0.9-3.fc38.x86_64
```

(I have a lot of JDKs installed)